test/e2e/flask/solana/common-solana.ts

```diff
@@ -67,6 +69,61 @@ export async function mockSolanaBalanceQuote(mockServer: Mockttp) {
       return response;
     });
 }
+
+export async function mockFungibleAssets(mockServer: Mockttp) {
+  return await mockServer
+    .forGet(`${SIMPLEHASH_URL}/api/v0/fungibles/assets`)
+    .thenCallback(() => {
+      return {
+        statusCode: 200,
+        json: {
+          fungible_id: 'solana.2RBko3xoz56aH69isQMUpzZd9NYHahhwC23A5F3Spkin',
+          name: 'PUMPKIN',
+          symbol: 'PKIN',
+          decimals: 6,
+          chain: 'solana',
+          previews: {
+              image_small_url: '',
+              image_medium_url: '',
+              image_large_url: '',
+              image_opengraph_url: '',
```

Comment:
leaving this intentionally empty, otherwise a subsequent request will be made to this urls